packages/@lwc/perf-benchmarks/package.json

```diff
@@ -29,7 +29,7 @@
         "@lwc/rollup-plugin-node-resolve-legacy": "npm:rollup-plugin-node-resolve@5.2.0",
         "folder-hash": "4.0.4",
         "markdown-table": "^3.0.3",
-        "tachometer": "0.5.10"
+        "tachometer": "0.7.0"
```

Comment:
We can't upgrade Tachometer. See the note above:

https://github.com/salesforce/lwc/blob/5a6188ec68471c1c2a7819d5bc7ac40fa19a287d/packages/%40lwc/perf-benchmarks/package.json#L15-L16